openwisp_controller/config/admin.py

```diff
@@ -52,6 +52,11 @@
 else:  # pragma: nocover
     from django.contrib.admin import ModelAdmin
 
+Device._meta.verbose_name = app_settings.get_settings_value(
```

Comment:
This will work and i think it is a better solution. But please define the setting where it was and use it as before
That is, define the `DEVICE_VERBOSE_NAME` in the setting file as you did it before and then here do
`Device._meta.verbose_name = app_settings.DEVICE_VERBOSE_NAME`

Reply:
``Device._meta.verbose_name = app_settings.DEVICE_VERBOSE_NAME``
is showing attribution error because the DEVICE_VERBOSE_NAME is not defined.

Should i use ``get_settings_value``
@NoumbissiValere 

Also i have to assign verbose_plural_name because it will not be automatically updated.

Reply:
@codesankalp please you have to defined `DEVICE_VRRBOSE_NAME` in config/settings file like you did it before [here](https://github.com/openwisp/openwisp-controller/pull/296#discussion_r499156526)

Reply:
According to your last commit, the `verbose_name_plural` depends on the `verbose_name` so any changes in your verbose name will be appropriately reflected in the plural

Reply:
@NoumbissiValere as I am changing the meta class externally it will not affect the plural one I have to manually add that too.

Reply:
I have checked your suggestion also but it didn't automatically update plurals.

Reply:
@codesankalp  this is the code you wrote;
```
Device._meta.verbose_name = app_settings.get_settings_value(
     'DEVICE_VERBOSE_NAME', 'Device'
)
Device._meta.verbose_name_plural = Device._meta.verbose_name + 's'
```
Right? If i'm wrong, please tell me but that's the code i see in this last commit.
The code works but it is against how we build things on `openwisp`.

I am saying you should define the settings in the file where other settings are defined. you did this before. [here](https://github.com/openwisp/openwisp-controller/pull/296#discussion_r499156526) Then come here and simple do;
`Device._meta.verbose_name = app_settings.DEVICE_VERBOSE_NAME`. 
This will mean that your total new changes in the admin file will look like this;

```
Device._meta.verbose_name = app_settings.DEVICE_VERBOSE_NAME
Device._meta.verbose_name_plural = Device._meta.verbose_name + 's'
```
Please if it is not still clear, let me know

Reply:
Thanks, @NoumbissiValere I did the same and I am testing it currently.
Previously I thought you want me to remove `verbose_name_plural` and use only `verbose_name`.
That's my confusion thanks for clearing it.